data/migrations/20211215202038_Submissions_Table.js

```diff
@@ -0,0 +1,23 @@
+exports.up = function (knex) {
+  return knex.schema.alterTable('Submissions-New', (t) => {
```

Comment:
alter Submissions table instead of Submissions-New. This is to anticipate the renaming of Submissions-New addressed in this PR: https://github.com/BloomTech-Labs/scribble-stadium-be/pull/96

Reply:
Thanks Chris, I'll make these changes shortly.